src/feature-test/user_sees_expanded_recipe_when_clicked.test.js

```diff
@@ -0,0 +1,53 @@
+import '../setupTest.js'
+import fetch from '../unit-test/mocks/recipeListFetch';
+
+describe('I.I.A.S', () => {
+  it('the user sees expanded recipe when clicking on title', async () => {
+    const browser = await puppeteer.launch({
+      // slowMo: 40,
+      // This two lines below allows the dev to see the flow on the chromium page
+      // headless: false,
+      // devtools: true
+    });
+    const page = await browser.newPage();
+    await page.goto('http://localhost:3000/');
+    await page.type('input[name="ing1"]', 'potatoes');
+    await page.type('input[name="ing2"]', 'sugar');
+    await page.type('input[name="ing3"]', 'milk');
+    await page.setRequestInterception(true);
+    await page.on('request', request => {
+      var url = request.url();
+      if(request.resourceType(fetch)) {
+        if (url === 'https://spoon-call.herokuapp.com/ingredients?id=6') {
+          request.respond({
+            body: JSON.stringify(
+              {extendedIngredients: [{id: 1, original: '1 Tbsp butter'}]}
+            )
+          })
+        } else if (url === 'https://spoon-call.herokuapp.com/instructions?id=6') {
+          request.respond({
+            body: JSON.stringify(
+              [{steps: [{number: 1, step: "preheat the oven to 200 C"}]}]
+            )
+          })
+        } else {
+            request.respond({
+              body: JSON.stringify(
+                [{id: 6, title: "Champ (Mashed Potatoes from Ulster)", image: "https://spoonacular.com/recipeImages/697254-312x231.jpg"}]
+              )
+            })
+        }
+      }
+    });
+    await page.click('input[type="submit"]');
+    page.on('response', response => { response.body })
+    await page.waitForSelector('a[href="recipe"]')
+    await page.click('a[href="recipe"]')
+    await page.waitForSelector('#recipe-expanded')
+    const found_instruction = await page.evaluate(() => window.find('preheat the oven to 200 C'));
+    expect(found_instruction).toBe(true)
+    const found_ingredient = await page.evaluate(() => window.find('1 Tbsp butter'));
+    expect(found_ingredient).toBe(true)
+    await browser.close();
+  },30000);
```

Comment:
This test is currently failing and so I can't check for sure but I don't think you need to increase the timeout for this test above the 5s default.

Reply:
Was not needed. Was used during test writing but should be removed for production. It's gone now.